### ISSUE TITLE
Not compatible with Keycloak 22

### ISSUE THREAD

**Issue by AndreVirtimo** [aerogear/keycloak-metrics-spi#168]
## Description
After the update to Keycloak 22.0.0 we can't use the metrics anymore.

I think the transition from Java EE to Jakarta EE is the cause (https://www.keycloak.org/2023/07/keycloak-2200-released.html).

## Expected Behavior
Working like before with Keycloak 21.

## Actual Behavior
When metrics API is called we get this error:
````
ERROR [io.quarkus.vertx.http.runtime.QuarkusErrorHandler] (executor-thread-17) HTTP Request to /auth/realms/master/metrics failed, error id: aa4af149-219c-4f56-9f95-b7a1d43e35e8-37: java.lang.NoClassDefFoundError: javax/ws/rs/WebApplicationException"
````

## Environment

Official Keycloak Docker container + keycloak-metrics-spi and a custom theme.

## Steps to reproduce
Start Keycloak 22 with keycloak-metrics-spi .

**Comment by logopk** [comment]
That's a me too

**Comment by diversit** [comment]
We're having the same issue. Working on a fix...

**Comment by diversit** [comment]
Provided a fix in #169

**Comment by pb82** [comment]
fixed in release 4.0.0: https://github.com/aerogear/keycloak-metrics-spi/releases/tag/4.0.0
